test/unit-test/test_restore.c

```diff
@@ -241,7 +241,7 @@ static void test_restore_list_backups_single_seed(void** state)
  */
 static void test_restore_list_backups_multiple_seeds(void** state)
 {
-    ListBackupsResponse list_backups_response;
+    ListBackupsResponse list_backups_response = {0};
```

Comment:
i see. the alternative (maybe more correct way) would be to not perform the follow up tests which inspect this struct in case it is not RESTORE_OK.

For my info, is tidy intelligent enough to not complain if you only access the struct in case of RESTORE_OK?

If so, it's a pity that cmocka's assert does not help tidy here.

Reply:
No I tried that and tidy still thinks that there can be an access of uninit memory.